iulianIachimov/src/main/java/datacollectiontask/Models/User.java

```diff
@@ -0,0 +1,26 @@
+package datacollectiontask.Models;
+
+import java.util.Map;
+
+public class User {
+    public long id;
+    public String username;
+    public Map<String, Address> addresses;
+
+    static long idCounter;
```

Comment:
before static should be the access modifiers, in case you leave empty, default it's used. You can use private as a better op[tion.